react/src/lib/components/Layout/Slider/slider.css

```diff
@@ -0,0 +1,24 @@
+.webviz-slider .rc-slider-track {
+    background-color: var(--menuLinkColor) !important;
+  }
+  
+  .webviz-slider .rc-slider-dot-active {  
+    border-color: var(--menuLinkHoverColor) !important;
+    border-color: var(--menuLinkHoverColor) !important;
+  }
+  
+  .webviz-slider .rc-slider-handle {
+    background-color: var(--menuLinkHoverColor) !important;
+    border-color: var(--menuLinkHoverColor) !important;
+  }
+  
+  .webviz-slider .rc-slider-handle-active:active {
+    border-color: var(--menuLinkHoverColor) !important;
+    box-shadow: 0 0 5px  var(--menuLinkHoverColor) !important;
+  }
+  
+  .webviz-slider .rc-slider-dot {
+    margin-left:-10px !important;
+    background-color: var(--menuBackground) !important;
+    border-color: var(--menuLinkHoverColor) !important;
+  }
```

Comment:
Is the `!important` here needed because `dash` set style directly?

Reply:
Yes. Ugly, but not sure there is another solution.